### PR TITLE
Use at::cpu in bench_approx

### PR DESCRIPTION
**Pull request by bertmaher**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#56563 Use at::cpu in bench_approx**

Differential Revision: [D27902737](https://our.internmc.facebook.com/intern/diff/D27902737)

### CODE REVIEW

File: benchmarks/cpp/tensorexpr/bench_approx.cpp [hunk 1]
```diff
@@ -366,7 +367,7 @@ static void tanh_caffe2(benchmark::State& state) {
   auto tanh = caffe2::TanhFunctor<caffe2::CPUContext>();
   at::tanh_out(A_t, B_ref);
```
Comment (navahgar) [inline benchmarks/cpp/tensorexpr/bench_approx.cpp:368]: Why not change this to `at::cpu::tanh_out` as well?

**Review by navahgar — APPROVED** [review]
Mostly LGTM. One comment below.

File: benchmarks/cpp/tensorexpr/bench_approx.cpp (hunk 1 above)
Comment (bertmaher) [reply benchmarks/cpp/tensorexpr/bench_approx.cpp:368]: Oh, I just didn't bother changing any of the ones that were there for reference (as opposed to for benchmarking).

**Review by ezyang — APPROVED** [review]
okey dokey

**Pull request closed without merging**